content/administration/upgrade.rst

```diff
@@ -1,12 +1,252 @@
+
+Process workflow
+================
+
+The upgrade process in a nutshell:
+
+#. You create a test upgrade request.
+#. Odoo processes the request automatically by running the database through an upgrade script, which
+   takes between 20 and 120 minutes.
+#. Odoo delivers a test database.
+#. You test your database for possible discrepancies (see :ref:`upgrade/test-guidance`)
+#. If there are any discrepancies, you report them to the Upgrade support team via the help portal
+   (see :ref:`upgrade/test-assistance`).
+#. We fix the issues and send you a new test database.
+#. Once you have completed the testing and are happy with the result, you decide on a date and time
+   when you stop users from accessing Odoo, freeze all data entries, and create an upgrade request
+   for the production upgrade.
+#. Odoo delivers the production database through the automated process.
+#. You restore it in your Production environment a few short hours later and continue working on the
+   newly upgraded database (this is done automatically on Odoo Online).
+
+.. _upgrade/testing-phase:
+
+Testing
+=======
+
+This phase allows you to review an upgraded version of your database without affecting your
+production database in any way. We suggest that you run the test upgrade process at least once, but
+you can do it as many times as you need (one at a time).
+
+Once you receive your upgraded test database, check that all data, processes, and functionality are
+still correct and working as expected.
+
+If you do find discrepancies, :ref:`report your issues <upgrade/test-assistance>` and :ref:`request
+a new test database <upgrade/test-db-request>` when the reported issues are fixed in the upgrade
+script.
+
+If you do not find any discrepancies, you can move on to the upgrade of your production database.
+
+.. important::
+   A test database is only intended for testing and remains completely unrelated to your present or
+   future production database. Any data you add, or changes you make, will not be reflected in your
+   upgraded production database.
+
+.. note::
+   Test databases are neutered, features are disabled to prevent them having impact on the
+   production database:
+
+   #. The serial number of the database is changed (to avoid it sending information as it was the
+      production database)
+   #. The base URL of the database is reset to `http://localhost:8069` and the email domain to
+      `localhost`
+   #. Scheduled actions are disabled (the calendar synchronization, the bank statement
+      synchronization, the planned automated actions, the fetching of incoming mail servers, etc.)
+   #. Outgoing mail servers are disabled, by archiving the existing ones and adding a
+      fake/non-working one
+   #. Payment providers and delivery carriers are reset to test environment
+   #. Accounting localization Electronic Data Interchange (EDI) services are disabled
+   #. A system parameter is set to tell the database has been neutered
```

Comment:
punctuation
- full stops
- no need for a comma between "Outgoing mail servers are disabled" and "by archiving ..."
```suggestion
   #. Scheduled actions are disabled (the calendar synchronization, the bank statement
      synchronization, the planned automated actions, the fetching of incoming mail servers, etc.).
   #. Outgoing mail servers are disabled by archiving the existing ones and adding a
      fake/non-working one.
   #. Payment providers and delivery carriers are reset to test environment.
   #. Accounting localization Electronic Data Interchange (EDI) services are disabled.
   #. A system parameter is set to tell the database has been neutered.
```